### ISSUE TITLE
1.16+ - Stronger Tornadoes

### ISSUE THREAD

**Issue by rankieth** [Mrbt0907/Weather2-Remastered#206]
It would be cool if tornadoes could grow to 1 mile (1609 blocks) or 2 miles (3218 blocks) wide. To get one of these large tornadoes, the environment has to be almost _perfect_, meaning that you'll probably have to live in an area that has dry, cold air meeting with warm, moist air.

Another cool idea would be if, on occasion, a tornado can have winds of 300+ mph. Technically, this is possible in 1.12 but you would have to get a tornado larger than an EF5 to spawn (which isn't realistic). 

![wedge1](https://github.com/Mrbt0907/Weather2-Remastered/assets/127988969/1cc4e3a3-e2d3-4a6e-92c6-623baca0c4d3)
![wedge2](https://github.com/Mrbt0907/Weather2-Remastered/assets/127988969/af61c8ef-3b2a-4643-b52c-ce26ed6d9e32)
![wedge3](https://github.com/Mrbt0907/Weather2-Remastered/assets/127988969/9bfd6965-58e5-4239-be34-ec2720506566)
![wedge4](https://github.com/Mrbt0907/Weather2-Remastered/assets/127988969/a0c5283b-b0cf-49aa-8b48-be10f3b499e6)

**Comment by CosmicHikari** [comment]
This is possible using the config system to enable storm max size to be higher (default is 300 blocks so we dont lag people out or make the block grabbing code die).

As for higher than 300mph wind speeds, considering you  mentioned 'realistic', is 300+mph possible in the real world?

**Comment by rankieth** [comment]
Yeah there’s been a few tornadoes that have reached wind speeds above 300. I’ve listed a couple examples.
1999 Bridgecreek-Moore
2024 Greenfield, Iowa

**Comment by CosmicHikari** [comment]
301-320 seems to be the upper limit, but that's only what we as humans know the limits to be. In nature, it's technically possible for a tornado's wind speed to reach the speed of sound, which would be horrifying. I like it. We'll discuss wind speed limits.
